.gitmodules

```diff
@@ -8,3 +8,6 @@
 	path = lib/sdl/SDL2
 	url = https://github.com/XboxDev/nxdk-sdl.git
 	branch = nxdk
+[submodule "lib/pdclib"]
+	path = lib/pdclib
+	url = https://github.com/XboxDev/nxdk-pdclib.git
```

Comment:
Wouldn't this check-out the wrong branch?

Reply:
git keeps a reference to the commit checked out in the submodule, so it still works. Specifying the branch seems to be helpful when updating the submodule, so I'll add it later tonight.